domains/chiller.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Website for Discord Bot - Chiller",
+  "owner": {
+    "username": "Code19Master",
+    "email": "divikbabbar@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://chiller.netlify.app"
```

Comment:
1. netlify doesnt work with is-a.dev, maybe try [railway](https://railway.app)?
2. cname shouldnt be a full url. (cc phenax)